mirrord/config/src/config.rs

```diff
@@ -76,6 +76,9 @@ pub enum ConfigError {
 
     #[error("Queue splitting config is invalid: {0}")]
     QueueSplittingVerificationError(#[from] QueueSplittingVerificationError),
+
+    #[error("Regex creation failed with: {0}")]
+    Regex(#[from] Box<fancy_regex::Error>),
```

Comment:
This error could be more specific, mentioning env mapping directly (and containing the bad regex)

Reply:
Improved a bit.